src/mem_table.rs

```diff
@@ -436,44 +494,56 @@ mod tests {
         let mut table = WritableKVTable::new();
 
         assert_eq!(table.table.size(), 0);
-        table.put(
-            Bytes::from_static(b"first"),
-            Bytes::from_static(b"foo"),
-            gen_attrs(1),
-        );
+        table
+            .put(
+                Bytes::from_static(b"first"),
+                Bytes::from_static(b"foo"),
+                gen_attrs(1),
+            )
+            .unwrap();
         assert_eq!(table.table.size(), 16); // first(5) + foo(3) + attrs(8)
 
         // ensure that multiple deletes keep the table size stable
         for ts in 2..5 {
-            table.delete(Bytes::from_static(b"first"), gen_attrs(ts));
+            table
+                .delete(Bytes::from_static(b"first"), gen_attrs(ts))
+                .unwrap();
             assert_eq!(table.table.size(), 13); // first(5) + attrs(8)
         }
 
-        table.put(
-            Bytes::from_static(b"abc333"),
-            Bytes::from_static(b"val1"),
-            gen_attrs(1),
-        );
+        table
+            .put(
+                Bytes::from_static(b"abc333"),
+                Bytes::from_static(b"val1"),
+                gen_attrs(6),
```

Comment:
note that I had to change the timestamps in these rows since these inserts were being added out of order